internal/version/version.go

```diff
@@ -17,12 +17,12 @@ var (
 	//
 	// Version must conform to the format expected by
 	// github.com/hashicorp/go-version for tests to work.
-	Version = "0.5.0"
+	Version = "0.6.0"
```

Comment:
Just something I noticed. We want changes on `main` to reflect the _next_ release, even if we don't intend to release this project again